rules.d/60-infiniband.rules

```diff
@@ -0,0 +1,6 @@
+# do not edit this file, it will be overwritten on update
+
+ACTION!="remove", SUBSYSTEM=="infiniband_verbs", IMPORT{builtin}="path_id"
+
+ENV{ID_PATH}=="?*", KERNEL=="uverbs*", SYMLINK+="infiniband/by-path/$env{ID_PATH}"
+ENV{ID_PATH}=="?*", KERNEL=="uverbs*", SYMLINK+="infiniband/by-ibdev/uverbs-$attr{ibdev}"
```

Comment:
Please add a generic guard for "remove" action and non-infiniband devices at the beginning, something like the following:
```
ACTION=="remove", GOTO="infiniband_end"
SUBSYSTEM!="infiniband_verbs", GOTO="infiniband_end"
KERNEL!="uverbs*", GOTO="infiniband_end"
```

Reply:
Done.